preview/galleon-local/pom.xml

```diff
@@ -66,6 +78,17 @@
             <artifactId>hibernate-search-mapper-orm-outbox-polling</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.wildfly.clustering</groupId>
+            <artifactId>wildfly-clustering-session-spec-servlet-6.1</artifactId>
```

Comment:
I don't see any use of this artifact anywhere.

I assume it's not listed in any boms/* pom because of a wildfly-clustering bom import somewhere in there, but to do anything at runtime there'd need to be a module.xml somewhere.

Reply:
Ah, good catch. I recall when I had originally done this at the beginning of the year, @pferraro had me add this. It's possible we do need a new clustering module too. Any clues Paul?

Reply:
New module added

Reply:
Without that module I think you should drop this dependency. It seems like cruft and WFLY-21088 should track the entire task of updating the clustering modules.

Logically the same would apply to the various 'isInvalid' changes in this PR. I don't have a strong opinion about that though. If those allow us to run and pass the TCK there is value in them, and they conceptually relate to a key goal of this PR. Of course if @pferraro says the changes are wrong or he doesn't think they should go in as part of this then that wins.